hieradata/dev/wso2/wso2is/pattern-4/default.yaml

```diff
@@ -0,0 +1,374 @@
+  slo_host_name_verifi_enabled: true
+
+wso2::passiveSTS:
+  slo_host_name_verifi_enabled: true
+
+wso2::evenet_listeners:
+  user_store_action_listener: true
+  identity_mgt_event_listener: false
+  identity_governance_event_listener: true
+  scimuser_operation_listener: true
+  identity_store_event_listener: true
+  daslogin_data_publisher: false
+  dassession_data_publisher: false
+  authn_data_publisher_proxy: true
+
+wso2::cache:
+  appauth_framework_session_context: false
+  authentication_context_cache    : false
+  appauth_framework_session_contex: false
+  authentication_request_cache : false
+  app_info_cache : false
+  authorization_grant_cache : false
+  oauth_cache : false
+  oauth_session_data_cache : false
+  samalsso_participant_cache : false
+  samalsso_session_index_cache : false
+  samalsso_session_data_cache : false
+  service_provider_cache: true
+  provisioning_connector_cache : true
+  provisioning_entity_cache : false
+  service_provider_provisioning_connector_cache: false
+  idP_cache_byauth_property : true
+  idp_cache_byhri : true
+  idp_cache_by_name: true
+
+wso2::tomcat:
+ maxThreads : 500
+ acceptCount : 500
+
+wso2::userstore_config:
+ userstore : <UserStoreManager class="org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager">
+             <Property name="TenantManager">org.wso2.carbon.user.core.tenant.JDBCTenantManager</Property>
+             <Property name="ReadOnly">false</Property>
+             <Property name="ReadGroups">true</Property>
+             <Property name="WriteGroups">true</Property>
+             <Property name="UsernameJavaRegEx">^[\S]{3,30}$</Property>
+             <Property name="UsernameJavaScriptRegEx">^[\S]{3,30}$</Property>
+             <Property name="UsernameJavaRegExViolationErrorMsg">Username pattern policy violated</Property>
+             <Property name="PasswordJavaRegEx">^[\S]{5,30}$</Property>
+             <Property name="PasswordJavaScriptRegEx">^[\S]{5,30}$</Property>
+             <Property name="PasswordJavaRegExViolationErrorMsg">Password length should be within 5 to 30 characters</Property>
+             <Property name="RolenameJavaRegEx">^[\S]{3,30}$</Property>
+             <Property name="RolenameJavaScriptRegEx">^[\S]{3,30}$</Property>
+             <Property name="CaseInsensitiveUsername">false</Property>
+             <Property name="SCIMEnabled">true</Property>
+             <Property name="IsBulkImportSupported">false</Property>
+             <Property name="PasswordDigest">SHA-256</Property>
+             <Property name="StoreSaltedPassword">true</Property>
+             <Property name="MultiAttributeSeparator">,</Property>
+             <Property name="MaxUserNameListLength">100</Property>
+             <Property name="MaxRoleNameListLength">100</Property>
+             <Property name="UserRolesCacheEnabled">true</Property>
+             <Property name="UserNameUniqueAcrossTenants">false</Property>
+             </UserStoreManager>
```

Comment:
Is this being correctly parsed for Puppet Hiera lookup?

Reply:
Yes